lib/stripe/errors.rb

```diff
@@ -27,6 +28,13 @@ def initialize(message = nil, http_status: nil, http_body: nil,
       @json_body = json_body
       @code = code
       @request_id = @http_headers[:request_id]
+      @error = construct_error_object
+    end
+
+    def construct_error_object
+      return nil if @json_body.nil? || !@json_body.key?(:error)\
```

Comment:
The `\` in the end could be removed 😅

Reply:
Haha, Juanito what are you doing here?! Thanks for the suggestion though — just pushed a commit directly in a1249af to remove it.

Reply:
My bad, I fat-fingered this `\` while fixing a Rubocop warning. Thanks for noticing @JuanitoFatas and for fixing @brandur-stripe!